src/activity_actor.cpp

```diff
@@ -5731,10 +5732,19 @@ void firstaid_activity_actor::finish( player_activity &act, Character &who )
         return;
     }
 
-    // TODO: Store the patient somehow, retrieve here
-    Character &patient = who;
+    Character *patient = patientID == get_avatar().getID() ? &get_avatar() :
+                         dynamic_cast<Character *>( g->find_npc( patientID ) );
+    if( !patient ) {
+        who.add_msg_if_player( m_bad,
+                               _( "Your patient can no longer be found so you stop using the %s." ),
+                               name );
```

Comment:
Does this happen in regular play? If not, I'd probably make this a debug message.

Reply:
It should rarely if ever happen. Mostly it is there as a safeguard from trying to use a nullptr. I can easily change it to a debug message - have to do some clang appeasement anyway.

Reply:
I imagine it'd happen mostly as a result of faulty saving/loading the activity or characters, in which case a debug message would be more appropriate.